server/routers/teams.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express')
+const router = express.Router();
+const context = require('../entities/database/context')
+const Team = context.Team
+
+router.get('/get', async (req, res) => {
+
+    // get the team id
+    let name = req.body.name
+    const team = await Team.findOne({
+        where: {
+            name
+        }
+    })
+    if (team === null) {
+        res.status(400).send({ message: "Team not found" })
+    }
+    else {
+        res.status(200).send({ team })
+    }
+
+})
+router.get('/getAll', async(req,res)=>{
```

Comment:
I would rather we ditch `get` for gets, using just `/`. I also think this should be merged with the `/get` route and the name should be given in the URL query.